src/main/java/com/google/sps/listeners/ConnectionPoolContextListener.java

```diff
@@ -0,0 +1,112 @@
+/*
+ * Copyright 2018 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.sps.listeners;
+
+import com.google.sps.classes.Utility;
+import com.zaxxer.hikari.HikariConfig;
+import com.zaxxer.hikari.HikariDataSource;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.SQLException;
+import javax.servlet.ServletContext;
+import javax.servlet.ServletContextEvent;
+import javax.servlet.ServletContextListener;
+import javax.servlet.annotation.WebListener;
+import javax.sql.DataSource;
+
+@WebListener("Creates a connection pool that is stored in the Servlet's context for later use.")
+public class ConnectionPoolContextListener implements ServletContextListener {
+
+  // Saving credentials in environment variables is convenient, but not secure - consider a more
+  // secure solution such as https://cloud.google.com/kms/ to help keep secrets safe.
+  private static final String 
+      CLOUD_SQL_CONNECTION_NAME = "internship-platform-step-2020:us-central1:mintern-instance";
+  private static final String DB_USER = "root";
+  private static final String DB_PASS = "mintern";
+  private static final String DB_NAME = "Mintern";
+
+  private DataSource createConnectionPool() {
+    // [START cloud_sql_mysql_servlet_create]
```

Comment:
What is this comment for?

Reply:
Came with the code I got from the example we based ourselves from, and I forgot to erase since we weren't using that specific part. (https://github.com/GoogleCloudPlatform/java-docs-samples/blob/master/cloud-sql/mysql/servlet/src/main/java/com/example/cloudsql/ConnectionPoolContextListener.java).